clang/include/clang/Sema/Sema.h

```diff
@@ -375,6 +375,9 @@ class SYCLIntegrationHeader {
 
     SourceLocation KernelLocation;
 
+    /// Whether this kernel is an ESIMD one.
+    bool IsESIMD;
```

Comment:
How about
```suggestion
    bool IsESIMDKernel;
```
To make it clear that this flag is about kernel and not language options, for example.

Reply:
ok